onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1144,7 +1144,7 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               OPTIONAL_VALUE)
         .Input(0,
                "input",
-               "3D tensor with shape (batch_size, sequence_length, hidden_size)",
+               "3D tensor with shape (batch_size, sequence_length, hidden_size) or 4D with shape (batch_size, num_heads, sequence_length, head_size)",
```

Comment:
Since you updated the operator spec, you need update two documents in this pull request.

You can download latest document from the failed kernel doc pipeline job: https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1220176&view=artifacts&pathAsName=false&type=publishedArtifacts